### PR TITLE
ADD:Unit test with jest added

### PR DESCRIPTION
**Pull request by maxbehzadi82**

### CODE REVIEW

File: launcher/package.json [hunk 1]
```diff
@@ -42,29 +48,22 @@
...
-  "jest": {
-    "collectCoverage": true,
-    "collectCoverageFrom": [
-      "src/**/*.{js,jsx}"
-    ],
-    "coverageReporters": [
-      "json",
-      "html"
-    ]
-  },
```
[the remark this answers is not in the document]
Comment (stefa2k) [reply launcher/package.json:67]: I think it's already there https://github.com/stereum-dev/ethereum-node/pull/37/files/fb1140570ecbdb491e6e878dd94b7e6b256c6433#diff-c8ff954df690199ba6aec7c4b734d77b311bd4088cdbb057965cc54f22eebc0a isn't this enough?

[the remark this answers is not in the document]
Comment (maxbehzadi82) [reply launcher/package.json:67]: It's already added.

[the remark this answers is not in the document]
Comment (NeoPlays) [reply launcher/package.json:67]: Well i couldn't find any coverage settings or am i missing something?

[the remark this answers is not in the document]
Comment (maxbehzadi82) [reply launcher/package.json:67]: Did u check the link which is shared by Stefan?

[the remark this answers is not in the document]
Comment (stefa2k) [reply launcher/package.json:67]: Max, there is no coverage report configured, or is it automatic?

[the remark this answers is not in the document]
Comment (NeoPlays) [reply launcher/package.json:67]: Yes i did, I mean not like Coverage is somethig game changing but I was just wondering.

[the remark this answers is not in the document]
Comment (maxbehzadi82) [reply launcher/package.json:67]: Seems,we do not need coverage report in vue3-jest config file. I couldn't find any infos.

[the remark this answers is not in the document]
Comment (stefa2k) [reply launcher/package.json:67]: this should be it: https://jestjs.io/docs/configuration#collectcoverage-boolean

[the remark this answers is not in the document]
Comment (maxbehzadi82) [reply launcher/package.json:67]: Solved

File: launcher/tests/unit/UI/LangDialog.spec.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { mount } from '@vue/test-utils'
+import LangDialog from "../../../src/components/UI/LangDialog";
+
+
+describe('LangButton.vue',()=>{
```
Comment (NeoPlays) [inline launcher/tests/unit/UI/LangDialog.spec.js:5]: `describe()` is used to group several related tests. You can also use it do make it easier to tell your tests apart if they have similar names.
You used `describe('LangButton.vue',()=>{` in **LangDialog.spec.js** you might want to change it to 
`describe('LangDialog.vue',()=>{`.

**Review by NeoPlays — CHANGES_REQUESTED** [review]
Code looks fine to me :thumbsup:  
Just change some names of tests i commented on

File: launcher/tests/unit/UI/LangDialog.spec.js (hunk 2 above)
Comment (maxbehzadi82) [reply launcher/tests/unit/UI/LangDialog.spec.js:5]: Thank you for being careful.

File: launcher/tests/unit/layers/BaseButton.spec.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { mount } from '@vue/test-utils'
+import BaseButton from "../../../src/components/layers/BaseButton";
+test('BackgroundPage contains img tag',()=>{
```
[the remark this answers is not in the document]
Comment (maxbehzadi82) [reply launcher/tests/unit/layers/BaseButton.spec.js:3]: I think test name is already deprecated.

[the remark this answers is not in the document]
Comment (maxbehzadi82) [reply launcher/tests/unit/layers/BaseButton.spec.js:3]: Oh I got what u mean :)

**Review by NeoPlays — APPROVED** [review]

**Pull request merged**